test/parallel/test-crypto-dh.js

```diff
@@ -307,6 +308,30 @@ if (availableCurves.has('prime256v1') && availableCurves.has('secp256k1')) {
   });
 }
 
+// Use of invalid keys was not cleaning up ERR stack, and was causing
+// unexpected failure in subsequent signing operations.
+if (availableCurves.has('prime256v1') && availableHashes.has('sha256')) {
+  const curve = crypto.createECDH('prime256v1');
+  const invalidKey = Buffer.alloc(65);
+  invalidKey.fill('\0');
+  curve.generateKeys();
+  try {
+    curve.computeSecret(invalidKey);
+    assert(false, 'key generation should have failed');
```

Comment:
This throws an exception so you still end up in the catch block.  The pattern tests usually use is this:
```js
let gotError = false;
try {
  // ...
} catch (e) {
  gotError = true;
  // ...
}
assert(gotError);
```